packages/marko-web-p1-events/components/track-native-story.marko

```diff
@@ -4,8 +4,35 @@ $ const { site } = out.global;
 $ const { story } = input;
 $ const config = site.getAsObject("p1events");
 $ const nxConfig = site.get("nativeX");
+$ const mindfulStoriesConfig = site.get("mindfulStories");
```

Comment:
I would change this to a more "generic" `mindful` object config, and change to `site.get("mindful.adsEnabled")` -- as this can be used more broadly to signify that the advertising functionality of mindful is active for the tenant.

Also, move the namespace to the `mindful.namespace` config value.